src/pipeline_migration/actions/migrate.py

```diff
@@ -614,6 +614,8 @@ def resolve(self, tb_upgrades: list[TaskBundleUpgrade]) -> None:
             for future in as_completed(futures):
                 try:
                     future.result()
+                except InvalidVersion as e:
+                    logger.warning(f"Skipping bundle upgrade due to invalid version: {e}")
```

Comment:
Please include the old and new bundle reference in this message. Refer to the error message below in block `except Exception as e`.